tests/typecheck/managers/querysets/test_from_queryset.yml

```diff
@@ -847,6 +847,41 @@
                     self.ttl = ttl
                     super().__init__(*a, **k)
 
+-   case: test_from_queryset_with_deferral
+    main: |
+        from myapp.models import Concrete
+        reveal_type(Concrete.objects.qs_meth())  # N: Revealed type is "builtins.int"
+        reveal_type(Concrete.objects.manager_meth())  # N: Revealed type is "builtins.str"
+    mypy_config: |
+        [mypy.plugins.django-stubs]
+        django_settings_module = myapp
```

Comment:
this is needed to skip the importing of models -- since the forced-deferral trick `models.py` triggers a `NameError` when actually imported